alibi/explainers/backends/cfrl_tabular.py

```diff
@@ -491,7 +503,7 @@ def get_inv_preprocessor(x_ohe: np.ndarray):
 
         np_X_inv = np_X_inv[:, inv_perm].astype(object)
         for i, fn in enumerate(feature_names):
-            type = feature_types[fn] if fn in feature_types else int
+            type = feature_types[fn] if fn in feature_types else float
```

Comment:
What was the reason for the int -> float change?

Reply:
I considered `float` to be more generic than `int`